src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, EventEmitter, Input, OnChanges, Output, OnInit } from '@angular/core';
+import { Exercise, ExerciseType, getIcon, getIconTooltip } from 'app/entities/exercise.model';
```

Comment:
`'ExerciseType' is defined but never used`

Reply:
Done